### DISCUSSION TITLE
Support JSON schema definitions

### DISCUSSION THREAD

**Discussion by margalit** [orval-labs/orval#242] [Q&A]
Hello! Lovely library.

I noticed you expect all external `$ref` pointers be OpenAPI files. Is there any way to support JSON schemas instead? Currently I get this error:

```
=> S2OError: Unsupported swagger/OpenAPI version: undefined
🛑  <PROJECTNAME> - Error: Unresolved $ref
```

FWIW, similar projects like [rtk-query-codegen](https://github.com/rtk-incubator/rtk-query-codegen) seem to support JSON schemas `$ref`s out of the box.

Cheers

**Comment by anymaniax** [comment]
Hello, from what I saw that should work the same way. Do you have an example of specification that you can provide?

**Reply by margalit** [reply]
Here's a minimal example that fails: https://github.com/margalit/orval-json-schema

Thank you for your help!
